mdpow/tests/test_dihedral.py

```diff
@@ -37,25 +37,25 @@ def teardown(self):
         self.tmpdir.dissolve()
 
     def test_dataframe(self):
-        dh1 = self.Ens.select_atoms('name C4 or name C17 or name S2 or name N3')
+        dh1 = self.Ens.select_atoms('name C4', 'name C17', 'name S2', 'name N3')
```

Comment:
revert changes, will be fixed in PR #218

Reply:
with 'or' vs. ', ' the selection string is reordered alphanumerically, dihedral may be calculated incorrectly if atom group is reordered

Reply:
My point was that there should not be duplicated code across PRs. **One PR solves ONE problem.** is a good mantra. I would have expected you to just `git revert <SHA1 commit ID>` the changes so that this PR would have retained the incorrect ordering and reference values. As soon as PR #218 was merged, we would have used the ''update this PR with current develop'' and merge develop with the good changes to test_recursive2() into this PR. I did this anyway and and the merge produced clean code without conflicts (so when you look into the file changes for this PR it appears as if you never touched test_recursive2() in this PR), but that's not always guaranteed.

In short: avoid duplicating code across PRs. If you need something for a certain PR to work then that's a sign that all effort needs to go into making this other thing work.

Reply:
My apologies, I think I tried to merge the branches and did so locally, but did not realize I could not merge and close the other branch remotely without the review being addressed, which resulted in duplicated code changes across branches after pushing the changes. I will continue working to close the gap in my knowledge of proper use of git when dealing with multiple related branches.